feat(node): Add more detailed OTEL docs (#10872)
getsentry/sentry-docs: docs/platforms/javascript/common/opentelemetry/required-instrumentation.mdx

  - javascript.azure-functions
  - javascript.connect
  - javascript.express
  - javascript.fastify
  - javascript.gcp-functions
  - javascript.hapi
  - javascript.koa
  - javascript.nestjs
  - javascript.sveltekit
  - javascript.astro
  - javascript.remix
notSupported:
  - javascript
---

By default, Sentry will register a series of OpenTelemetry instrumentation to automatically capture spans for things spanning incoming and outgoing HTTP requests, DB queries and more.

If tracing is not enabled (no `tracesSampleRate` is defined in the SDK configuration), only a minimal amount of OpenTelemetry instrumentation will be registered. This includes the following:

- <PlatformLink to="/configuration/integrations/http/">httpIntegration</PlatformLink> registers [@opentelemetry/instrumentation-http](https://www.npmjs.com/package/@opentelemetry/instrumentation-http)
- <PlatformLink to="/configuration/integrations/nodefetch/">nativeNodeFetchIntegration</PlatformLink> registers [opentelemetry-instrumentation-fetch-node](https://www.npmjs.com/package/opentelemetry-instrumentation-fetch-node)

<PlatformSection notSupported={['javascript.aws-lambda', 'javascript.gcp-functions']}>

>   <Note>
>     If tracing is not enabled, performance instrumentations will not be registered but they will still be included in the bundle. If you
>     want to reduce the bundle size or used dependencies, you can also <PlatformLink to="/configuration/tree-shaking/#setting-up-sentry-without-performance-integrations">Set up Sentry without Performance Integrations</PlatformLink>.
>   </Note>

Reviewer: ```suggestion
  <Note>
If tracing hasn't been enabled, performance instrumentations won't be registered, but performance will still be included in your bundle. If you want to reduce the bundle size or the amount of dependencies used, you can <PlatformLink to="/configuration/tree-shaking/#setting-up-sentry-without-performance-integrations">tree shake</PlatformLink> to set up Sentry without Performance Integrations.
  </Note>
```
I'm going to set up a few minutes for us to chat. I just want to make sure I understand this correctly with the new updates we just made as a company (moving from performance to tracing).

  - javascript.aws-lambda
  - javascript.azure-functions
  - javascript.connect
  - javascript.express
  - javascript.fastify
  - javascript.gcp-functions
  - javascript.hapi
  - javascript.koa
  - javascript.nestjs
  - javascript.sveltekit
  - javascript.astro
  - javascript.remix
notSupported:
  - javascript
---

By default, Sentry will register a series of OpenTelemetry instrumentation to automatically capture spans for things spanning incoming and outgoing HTTP requests, DB queries and more.

If tracing is not enabled (no `tracesSampleRate` is defined in the SDK configuration), only a minimal amount of OpenTelemetry instrumentation will be registered. This includes the following:

- <PlatformLink to="/configuration/integrations/http/">httpIntegration</PlatformLink> registers [@opentelemetry/instrumentation-http](https://www.npmjs.com/package/@opentelemetry/instrumentation-http)
- <PlatformLink to="/configuration/integrations/nodefetch/">nativeNodeFetchIntegration</PlatformLink> registers [opentelemetry-instrumentation-fetch-node](https://www.npmjs.com/package/opentelemetry-instrumentation-fetch-node)

<PlatformSection notSupported={['aws-lambda', 'gcp-functions']}>
  <Note>

>     Note that even though all performance instrumentations will not be registered
>     if tracing is not enabled, it will still be included in the bundle. If you

Reviewer 2: l: I suggest we shorten this a bit:

```suggestion
    If tracing is not enabled, performance instrumentations will not be registered bit they will still be included in the bundle. If you
```
